src/services/identity/UsersService.ts

```diff
@@ -79,8 +82,28 @@ class UsersService {
         },
       ],
     })
+  }
 
-    return !!siteMember
+  async isSiteAdmin(userId: string, siteName: string) {
```

Comment:
Same for this method, should return a boolean instead of a User object.

Reply:
See above

Reply:
turns out this method isn't used at all? more inclined to delete but renamed for now